app/component/itinerary/NaviCardContainer.js

```diff
@@ -37,17 +36,9 @@ function NaviTop(
   const destCountRef = useRef(0);
 
   const handleClick = () => {
-    setShowMessages(!showMessages);
+    setCardExpanded(!cardExpanded);
   };
 
-  useEffect(() => {
-    const timer = setTimeout(() => {
-      setShowMessages(false);
-    }, 5000);
-
-    return () => clearTimeout(timer);
-  }, []);
-
   useEffect(() => {
```

Comment:
This looks like a good candidate for a custom hook as the logic inside `useEffect` is all in all quite multifaceted.

Reply:
Yeah, this needs to be refactored, but I think it would be more suited for seperate PR